omnimove_external_control/include/omnimove/omnimove_external_control.hpp

```diff
@@ -0,0 +1,52 @@
+#ifndef OMNIMOVE_CONTROLLER_HPP
+#define OMNIMOVE_CONTROLLER_HPP
+
+#include <rcl/logging.h>
+#include <boost/asio.hpp>
+#include <boost/chrono.hpp>
+#include <boost/circular_buffer.hpp>
+#include <hardware_interface/handle.hpp>
+#include <hardware_interface/system_interface.hpp>
+#include <omnimove/external_control_message.hpp>
+namespace omnimove
+{
+class OmnimoveExternalControl : public hardware_interface::SystemInterface
```

Comment:
class should be rather `KukaOmnimoveHardwareInterface` to align to conventions
file name for header and source should be hardware_interface.h/.cpp
include directory name should match the package name

Reply:
renamed file. 
Regarding the name to .h. I think it is fine to name cpp header file as hpp. That seems to be convention in ROS